### ISSUE TITLE
Tab Order Visualization appears to be missing

### ISSUE THREAD

**Issue by mmoineau** [ThePacielloGroup/ARC-Toolkit#17]
It appears the ability to turn on Tab Order Visualization didn't seem to be working any longer (in ARC Toolkit Version 5.3.0). 
Is there a way to show the tab order as we could in the older version of ARC Toolkit?

You can see the feature as shown in your own video on YouTube:
https://www.youtube.com/watch?v=FmIzRdLBOmA&t=250s

**Comment by poorgeek** [comment]
@mmoineau, this feature will available in the next release of ARC Toolkit which is going through our testing process now.

Duplicate of #12
